src/openqa_client/client.py

```diff
@@ -377,3 +377,33 @@ def get_jobs(
             # there as it only considers the jobs queried.
             jobdicts = self.find_clones(jobdicts)
         return jobdicts
+
+    def get_latest_build(
+        self,
+        group_id: int,
+        all_passed: bool = True,
+        sorted_key: Callable = float,
+        default_return: str = "0",
+    ) -> str:
+        """Identify latest build number in target Job Group
+
+        Args:
+            group_id (int): Job Group ID
+            all_passed (bool, optional): Controls whether just
+                last one will be selected or last with all jobs passed. Defaults to True.
+            sorted_key (Callable, optional): To judge about latest build we need to order them.
+                This is callback which will be used in sorted function to sort builds.
+                Defaults to float.
+            default_return (str, optional): In case all available builds have not acceptable format
+                or there is no builds in this job group this value identofy what to return. Defaults to "0".
+
+        Returns:
+            str: string representation of last BUILD
+        """
+        resp = self.openqa_request("GET", f"/group_overview/{group_id}.json")
+        if all_passed:
+            passed = [r for r in resp["build_results"] if r["all_passed"] == 1]
+            builds = [r["build"] for r in passed if r["build"].isdigit()]
+        else:
+            builds = [r["build"] for r in resp["build_results"] if r["build"].isdigit()]
+        return list(sorted(builds, key=sorted_key))[-1] if builds else default_return
```

Comment:
This should in theory use `max` instead of `sorted` but it looks like no custom comparison function can be specified when using `max`. So I suppose `sorted` is the way to go.